#!/usr/bin/env python
PACKAGE = "moveit_shadow_test"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("numero_dedos",    int_t,    0, "Numero de dedos que se usan", 5,  2, 5)
gen.add("min_pressure_threshold", double_t, 0, "Umbral minimo de presion", 2.0, 0.5, 3.0)
gen.add("max_pressure_threshold", double_t, 0, "Umbral maximo de presion", 5.0, 3.0, 7.0)

exit(gen.generate(PACKAGE, "moveit_shadow_test", "grasp_reconfigure"))